Tests/CryptoSwiftTests/ExtensionsTest.swift

```diff
@@ -60,7 +60,9 @@ final class ExtensionsTest: XCTestCase {
         let ii:Int = 21
         XCTAssert(ii &<< 1 == ii << 1, "shift left failed")
         XCTAssert(ii &<< 8 == ii << 8, "shift left failed")
-        XCTAssert(ii &<< ((MemoryLayout<Int>.size * 8) - 1) == ii << ((MemoryLayout<Int>.size * 8) - 1), "shift left failed")
```

Comment:
eh, Swift 3 compiler getting better and better

![screen shot 2016-09-08 at 11 07 47](https://cloud.githubusercontent.com/assets/758033/18343844/57524d00-75b5-11e6-9f41-05e3236ee2b1.png)